docs_src/multiple_values/options_with_multiple_values/tutorial002.py

```diff
@@ -0,0 +1,25 @@
+from enum import Enum
```

Comment:
A quick note that this tutorial file isn't actually referenced from anywhere within the tutorial markdown files.